### ISSUE TITLE
Show freeway exits with the straight arrow that the driver actually sees

### ISSUE THREAD

**Issue by kevinp2** [osmandapp/OsmAnd#12768]
# 🚀 feature request

### Description

In the US and Canada, freeway exits are marked with a straight arrow pointing to the right at about a 45 degree angle.

![image](https://user-images.githubusercontent.com/609677/133349150-469d2c4e-1786-4f8a-9f32-6b243af5704e.png)

OSMAnd shows this with a curving arrow that is not intuitive and doesn't match the actual road sign.

![image](https://user-images.githubusercontent.com/609677/133349170-225ff984-3050-4104-9fe2-d05d1672d544.png)

### Describe the solution you'd like

Please show at least a straight arrow for a freeway exit.  As a bonus, it could be colored according to country scheme, e.g. white on blue for the EU, white on green for North America.

### Describe alternatives you've considered

None.  

The OSM node that is responsible for the pictures is here: https://www.openstreetmap.org/node/152384622#map=19/30.21152/-97.85814

**Comment by tradavyvy** [comment]
I would not needlessly complicate the depiction in Osmand (and the work of the programmers)
As a map-app osmand gives a schematic rendering of the real world and I think the yellow arrows do a good job at that. Besides a higway exit usualy has a straight exit-lane before it bends of to the right (or left).

**Comment by sonora** [comment]
Our arrows turn from
- gray - _mere pre-info about the next turn_   to
- yellow - _catching your attention for the next turn coming up soon_  to
- green - announcing the "_turn now_" situation.

In deed, as for the arrow shapes we use a generic pattern globally. Trying to mimic different signing habits in different regions would be a sizeable complication, even not yet considering if the signing habits are really used consistently, e.g. enforced by a national highway authority, or used rather flexibly on a case-by-case basis. In the latter case it may prove difficult or even impossible to reliably deduce the singing practice's ground truth from the underlying OSM map data. Hence my first guess is that the return of invest of an attempt to do something here may not be too good on a global scale ...

**Comment by sonora** [comment]
PS: Regarding above colors: When I implemented this many years ago, the only globally accepted convention I found was to assimilate the stop light ('traffic signal') logic where _yellow/amber_ catches your attention and _green_ means "Go". Other color schemes like e.g. using red to notify "turn now" resulted in some users getting confused wondering if red would not actually try to say "not here" or something ...

**Comment by tradavyvy** [comment]
I think the colorscheme works fine and is perfectly clear, for me it is anyway.

**Comment by kevinp2** [comment]
Hi folks, I agree that the coloring is secondary and a nice to have.

But the arrow shape should conform to what users see in the real world.  I did some digging and in my admittedly non-comprehensive search, **every single country uses a straight arrow at the point of the freeway exit**, and motorists recognize this sign as their last chance to take action.

**Comment by vshcherb** [comment]
Yes, we have a setting of driving region, may be it should be configured there.

**Comment by sonora** [comment]
It'll be difficult...

Here's some examples from the US and Germany, undoubtely on the more organized side of the planet... ;(.  As you can see, a wild mixturd of arrows pointing up, pointing down, 45 degrees, horizontal, or curved...

![proxy-image(4)](https://user-images.githubusercontent.com/845735/133506638-a58712d7-48b2-416e-81ff-0225778c3e5e.jpg)
![proxy-image(3)](https://user-images.githubusercontent.com/845735/133506642-4e6b697c-6efe-469a-9d98-5bda73011852.jpg)
![proxy-image](https://user-images.githubusercontent.com/845735/133506643-f0518872-66af-457b-91d2-cb74f16fbe56.png)
![proxy-image(2)](https://user-images.githubusercontent.com/845735/133506646-f3220023-4711-4536-87a3-09bc61ae176b.jpg)
![proxy-image(1)](https://user-images.githubusercontent.com/845735/133506647-cca2b232-249e-4b7a-accc-ab7a9d3a65df.jpg)
![proxy-image](https://user-images.githubusercontent.com/845735/133506649-c484dfbb-312c-4851-b7dd-e70220c81d9e.jpg)

**Comment by tradavyvy** [comment]
As I said before, keep it simple.
The current system works fine.
It is not the task of a map(-app) to emulate the real world, but to give a schematic representation in which (as much as posible) relevant information is given in a clear and easy to interpret way.

As an exagurated example:
Dirt roads in large parts of new mexico tend to be a yellowish gray color. In south utah they are often bright red. It will not make the map more usable if these colors showed up in the rendering.
It is however preferrable that it is clear that both are dirt roads (and maybe what the quality is)

**Comment by jfburke3** [comment]
I think the arrows should be left as they are.  The curved arrows accurately reflect the _actual path you take when you exit the highway_ and not the over-simplified straight-line path shown on the exit signs.
